packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -398,6 +442,16 @@ class BouncingScrollPhysics extends ScrollPhysics {
   @override
   double applyBoundaryConditions(ScrollMetrics position, double value) => 0.0;
 
+  @override
+  double adjustPositionForNewDimensions(ScrollMetrics position, {
+    @required bool isScrolling,
+    @required double velocity,
+  }) {
+    if (!isScrolling && velocity == 0.0)
```

Comment:
We shouldn't do comparisons of doubles, this should be `isClose(velocity, 0.0, tolerance:0.01)`.

Reply:
Will we ever have a velocity other than zero when we aren't scrolling?